src/displayapp/screens/Notifications.cpp

```diff
@@ -126,12 +129,31 @@ void Notifications::OnPreviewInteraction() {
   }
 }
 
+void Notifications::OnPreviewDismiss() {
+  notificationManager.Dismiss(currentId);
+  if (timeoutLine != nullptr) {
+    lv_obj_del(timeoutLine);
+    timeoutLine = nullptr;
+  }
+  currentItem.reset(nullptr);
+  dismissingNotification = true;
+  afterDismissNextMessageFromAbove = true; // show next message coming from below
+  app->SetFullRefresh(DisplayApp::FullRefreshDirections::RightAnim);
```

Comment:
could you refactor the copied code from the swipe right event handling into a new common helper function `dismissToBlackness()` (or a different meaningful name ;) )

Reply:
I'll create a common helper function